### ISSUE TITLE
Vision should make a positive statement of how the Web should evolve

### ISSUE THREAD

**Issue by jyasskin** [w3c/AB-public#215]
The [W3C’s Vision for the World Wide Web](https://www.w3.org/TR/w3c-vision/#vision-web) section contains 4 points that mostly describe ways to "not mess up", with no pointers toward what we _should_ be trying to do.

We should work toward having a W3C-wide positive vision for the web, but I think it's ok to publish this version (consensus on "don't break things" is a step forward), and then develop that direction for the next version.

**Comment by martinthomson** [comment]
The W3C produces technical standards.  It would be a shame to limit the vision to stuff about the _how_ to the detriment of the _what_.  There are a few hooks (like priority of constituencies) but not really a whole lot of substance about what it means to have a technical direction.

I do want to point out that it might be orders of magnitude easier to agree on the _how_ part than the _what_ part.  But that's where something like a vision really delivers value.  Operational aspirations are great, but exemplary execution is no good without an outcome.

(It seems like many Bond villains appear to have excellent organizational discipline, health care,  remuneration, etc...)

**Comment by martinthomson** [comment]
I consider this issue a blocker.  The TAG has discussed this and does not have consensus either way on that point.

I personally would prefer that this document not be presented as a vision, but as a set of operating principles for the consortium.  That is what it is.  If it is presented as a Vision, it will do harm to the institution that it is supposed to promote and strengthen.  The W3C is a technical standards development body and publishing a vision that does not include any substantive technical component risks undermining the credibility of the institution.

**Comment by chrishtr** [comment]
> We should work toward having a W3C-wide positive vision for the web...

Here's my proposal for the positive vision for what we should aim for. The web should be:

1. The best way to represent all of humanity's information, learning and wisdom.
2. The best way to easily express any user experience across all modalities, languages and cultures.
3. Universally available and accessible to all users and all computing devices.
4. An open system available to all that is better than any proprietary alternative.

**Comment by martinthomson** [comment]
I don't think that a true vision for the technical future of the web is so brief.  I refer to [Mozilla's vision](https://www.mozilla.org/en-US/about/webvision/full/), which is a substantial document.  I don't think that this is the right sort of thing for the W3C to advocate for as it is somewhat narrower in scope and maybe less positive than I would personally advocate for in 2024.

However, [as I mentioned](https://github.com/w3c/AB-public/issues/215#issuecomment-2520257238), the title of the document could more accurately reflect the somewhat internal focus of this document.  This is not a vision statement, but a set of guidelines for how we think the consortium should operate.

Hence: "Operating Principles for W3C" is my suggestion as a title.

**Comment by chrishtr** [comment]
Hi Martin, couple of questions:

> I don't think that a true vision for the technical future of the web is so brief.

Fair. I think it's worth having a brief and bold statement that everyone can get behind, but just my 2c.

**Comment by cwilso** [comment]
The "Vision of the WWW" section was always intended to be a very brief summary.  I suggested a long time ago removing this entire section from this Vision of the W3C, but was overruled.  I think a longer statement around the technical evolution of the Web would be a good idea; I think that's probably more in the remit of the TAG.  I think this is deferrable.

**Comment by fantasai** [comment]
Agree with @cwilso: this is the vision document for W3C, not for the Web. We might consider drawing up such a document (maybe as a joint AB+TAG deliverable?), but it would be a separate document.

**Comment by mgendler** [comment]
The editors and chair discussed, editor to discuss with Martin directly to figure out next steps.

**Comment by cwilso** [comment]
With #254 merged, I believe we are now okay to move this to "defer" to pursue further post-v1.  @martinthomson ?

**Comment by martinthomson** [comment]
After having spoken with @tantek, I was given to understand that more significant changes were afoot.  I've been busy with other tasks for a week or so, but #254 doesn't work for me.  I think that it needs to be more substantive than that.

**Comment by tantek** [comment]
At this point given the depth and breadth of review (and wide approval as tracked in #195) the document has had, more substantial changes like re-organizing sections (or completely rewriting section 3) was deemed out of scope in discussion with VisionTF chair and fellow editor. 

Since the TAG as a whole did not label this as a blocker (and there was no evidence that that had changed in any regard since the issue filing) we reduced the scope of section 3 accordingly (which itself had started to succumb to rewriting by committee due to the back/forth in #211) with the expectation that it will get updated for v1.1 of the Vision, and are leaving this issue open and labeled "defer" to address the remaining aspects and help drive that update.

At this point, one escalation path would be to take it back to the TAG to see if you can convince the TAG that this should be a blocker. Otherwise we are not expecting to make any more edits to this section before.

Lastly, the original issue as filed, 
> 4 points that mostly describe ways to "not mess up", with no pointers toward what we should be trying to do.

is fully resolved as all 4 points have been reworded as positive statements:
* The Web is for [all humanity](https://www.w3.org/TR/ethical-web-principles/#allpeople).
* The Web is designed for the [good of all people](https://www.w3.org/TR/ethical-web-principles/#noharm).
* The Web must be [safe to use](https://www.w3.org/TR/ethical-web-principles/#privacy).
* There is [one interoperable world-wide Web](https://www.w3.org/TR/ethical-web-principles/#oneweb).

And subsequent principles in the next section address the what of each of those more specifically.

Any further suggestions for changing section 3, or section 4, or re-ordering beyond the original issue description should be filed as a new issue.

**Comment by dontcallmedom** [comment]
(I think this issue aligns with what I had tried to convey in https://github.com/w3c/AB-public/issues/143 for future discussions)
